test/services/katello/registration_manager_test.rb

```diff
@@ -17,12 +17,17 @@ class RegistrationManager < RegistrationManagerTestBase
         @content_view_environment = katello_content_view_environments(:library_dev_view_library)
         @activation_key = katello_activation_keys(:library_dev_staging_view_key)
         @host_collection = katello_host_collections(:simple_host_collection)
+        @one_host_limit_host_collection = katello_host_collections(:one_host_limit_host_collection)
         @activation_key.host_collections << @host_collection
         @org = @library.organization
         @facts = {'network.hostname' => 'foo.example.com'}
         @host = FactoryBot.create(:host, organization: @org)
       end
 
+      after :all do
```

Comment:
Why? :o

Reply:
I noticed the test was creating multiple host collections in the DB from the same fixture, and adding them to `new_host`'s `host_collections`. just wanted to make sure that's cleaned up so there's no bleeding of anything to other tests!